### DISCUSSION TITLE
Attachment box at top/right in compose window

### DISCUSSION THREAD

**Discussion by davoper** [Aris-t2/CustomCSSforTb#19] [Ideas]
Would it be possible to use CustomCSSforTb to once again place Attachments in the top/right part of compose window like it was in previous TB versions?
Right now, having attachment list in the bottom of the mail composition window makes it really easy to overlook them before sending e-mail.

See picture from old TB (v68) for reference:
![image](https://github.com/Aris-t2/CustomCSSforTb/assets/5680741/a6030d48-c4c9-4c7d-a04c-c68227eafbdc)

**Comment by Aris-t2** [comment]
I did not find a working way to reorder compose windows content. Moving attachment box is something I wanted to achieve for a while now.
